source/components/patternfly/tests-src/_includes/widgets/communication/alert-danger-dismissable.html

```diff
@@ -0,0 +1,7 @@
+<div class="alert alert-danger alert-dismissable">
```

Comment:
I still have the original softlink.
Should I remove it? Or delete the files I added? They came with a prior rebase...